### PR TITLE
CICD 4347 Add Code Linter 

### PR DESCRIPTION
**Pull request by aaryan2134**
Implemented Code Linter and corrected issues.

Auto fix issues using "code formatter"
image
image

No issues found with "code linter"
image

### CODE REVIEW

**Comment by aaryan2134** [comment]
Please review this and let me know if any change is required

**Comment by PineappleIOnic** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Comment by aaryan2134** [comment]
Sure. Thank you!

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 Sorry, but it looks like there are conflicts now. Would you please resolve the conflicts and then we'll try to merge it in ASAP.

**Comment by aaryan2134** [comment]
Done. I have resolved the conflicts

**Review by stnguyen90 — APPROVED** [review]
Hey 👋 awesome work on your PR! We've approved your work and it'll be merged soon!

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
@aaryan2134 for some reason, the build is failing now. Something may have happened while you were fixing the merge conflicts. Would you please fix the builds? You might want to clear out all your changes and run pint on a clean master.

**Comment by aaryan2134** [comment]
I have removed all the changes made using code linter. But still the build is failing...

**Comment by aaryan2134** [comment]
I am not able to understand why this is not working... My build has been failing since the very first commit.. even without changing any files at all

File: composer.lock [hunk 1]
```diff
@@ -3447,16 +3518,16 @@
         },
         {
             "name": "vimeo/psalm",
-            "version": "4.27.0",
+            "version": "4.29.0",
```
Comment (stnguyen90) [inline composer.lock:3521]: I think the problem could be the update to this dependency. Perhaps if you install pint without updating psalm, there won't be any errors.

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

File: composer.lock (hunk 1 above)
Comment (aaryan2134) [reply composer.lock:3521]: Sure. Will make this change and let you know

**Comment by aaryan2134** [comment]
This worked finally! Thank you so much
That was something I couldn't have noticed ever :)

File: composer.lock (hunk 1 above)
Comment (aaryan2134) [reply composer.lock]: Done

**Review by stnguyen90 — APPROVED** [review]
Hey 👋 awesome work on your PR! We've approved your work and it'll be merged soon!

**Comment by stnguyen90** [comment]
@christyjacob4 could we please get this merged in ASAP to avoid merge conflicts?

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**